cmd/agent/subcommands/run/command.go

```diff
@@ -122,6 +122,7 @@ import (
 	"github.com/DataDog/datadog-agent/comp/remote-config/rcservice/rcserviceimpl"
 	"github.com/DataDog/datadog-agent/comp/remote-config/rcservicemrf/rcservicemrfimpl"
 	"github.com/DataDog/datadog-agent/comp/remote-config/rctelemetryreporter/rctelemetryreporterimpl"
+	remoteagentfx "github.com/DataDog/datadog-agent/comp/remoteagent/fx"
```

Comment:
I would move this component inside the `core` folder `comp/core/....`

Reply:
Sounds good, will do.